<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QParticleSelector</class>
 <widget class="QWidget" name="QParticleSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QParticleSelector</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="pb_1">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/1_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/1.png</normalon>:/Particles/ParticleIcons/1_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pb_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/2_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/2.png</normalon>:/Particles/ParticleIcons/2_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/3_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/3.png</normalon>:/Particles/ParticleIcons/3_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="pb_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/4_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/4.png</normalon>:/Particles/ParticleIcons/4_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="pb_5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/5_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/5.png</normalon>:/Particles/ParticleIcons/5_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="pb_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/6_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/6.png</normalon>:/Particles/ParticleIcons/6_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pb_7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/7_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/7.png</normalon>:/Particles/ParticleIcons/7_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pb_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/8_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/8.png</normalon>:/Particles/ParticleIcons/8_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pb_9">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/9_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/9.png</normalon>:/Particles/ParticleIcons/9_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pb_10">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/10_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/10.png</normalon>:/Particles/ParticleIcons/10_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pb_11">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/11_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/11.png</normalon>:/Particles/ParticleIcons/11_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pb_12">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/12_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/12.png</normalon>:/Particles/ParticleIcons/12_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pb_13">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/13_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/13.png</normalon>:/Particles/ParticleIcons/13_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pb_14">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/14_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/14.png</normalon>:/Particles/ParticleIcons/14_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pb_15">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/15_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/15.png</normalon>:/Particles/ParticleIcons/15_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pb_16">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/16_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/16.png</normalon>:/Particles/ParticleIcons/16_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QPushButton" name="pb_17">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/17_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/17.png</normalon>:/Particles/ParticleIcons/17_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="2" column="5">
       <widget class="QPushButton" name="pb_18">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="qparticleselector.qrc">
          <normaloff>:/Particles/ParticleIcons/18_fade.png</normaloff>
          <normalon>:/Particles/ParticleIcons/18.png</normalon>:/Particles/ParticleIcons/18_fade.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>120</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pb_1</tabstop>
  <tabstop>pb_2</tabstop>
  <tabstop>pb_3</tabstop>
  <tabstop>pb_4</tabstop>
  <tabstop>pb_5</tabstop>
  <tabstop>pb_6</tabstop>
  <tabstop>pb_7</tabstop>
  <tabstop>pb_8</tabstop>
  <tabstop>pb_9</tabstop>
  <tabstop>pb_10</tabstop>
  <tabstop>pb_11</tabstop>
  <tabstop>pb_12</tabstop>
  <tabstop>pb_13</tabstop>
  <tabstop>pb_14</tabstop>
  <tabstop>pb_15</tabstop>
  <tabstop>pb_16</tabstop>
  <tabstop>pb_17</tabstop>
  <tabstop>pb_18</tabstop>
 </tabstops>
 <resources>
  <include location="qparticleselector.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
